product/reports/680_Performance by Asset Type - Middleware Datasources/120_Datasource Pool - hourly for the last day.yaml

```diff
@@ -0,0 +1,89 @@
+--- 
+where_clause: 
+dims: 
+created_on: 2016-07-07 14:53:10.822035 Z
+reserved: 
+title: "Datasource Pool - hourly for the last day"
+conditions: 
+updated_on: 2016-07-08 17:23:33.732482 Z
+order: Descending
+graph: 
+menu_name: "Datasource Pool - hourly for the last day"
+rpt_group: Custom
+priority: 
+col_order: 
+- id
+- name
+- start_time
+- mw_ds_available_count_min
+- mw_ds_available_count_avg
+- mw_ds_available_count_max
+- mw_ds_in_use_count_min
+- mw_ds_in_use_count_avg
+- mw_ds_in_use_count_max
+- mw_ds_timed_out_min
+- mw_ds_timed_out_avg
+- mw_ds_timed_out_max
+- mw_ds_average_get_time_min
+- mw_ds_average_get_time_avg
+- mw_ds_average_get_time_max
+- mw_ds_average_creation_time_min
+- mw_ds_average_creation_time_avg
+- mw_ds_average_creation_time_max
+timeline: 
+id: 680
+file_mtime: 
+categories: 
+rpt_type: Custom
+filename: 
```

Comment:
@abonas Specify a unique filename and it should work as is. 

See [here](https://github.com/ManageIQ/manageiq/blob/fcc88eecc8886fd885ee9564de85a96719864580/app/models/miq_report/seeding.rb#L69)

Note, [it deletes an id value if it exists](https://github.com/ManageIQ/manageiq/blob/fcc88eecc8886fd885ee9564de85a96719864580/app/models/miq_report/seeding.rb#L54), but I don't think it makes sense to include the id field at all.